### PR TITLE
Changed some things and plugged loopholes

### PR DESCRIPTION
**Pull request by yet-another-account**
Made some things clearer; some typos still need to fixed

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -26,13 +30,16 @@ Version 2.0.3
 
 **Modpack Creator**: An individual who creates modpacks.
 
-**Author(s)**: Original programmer or programmers responsible for writing the Mod.
+**Author(s)**: Original programmer(s), texturer(s), and contributors responsible for creating, in part or in whole, the Mod.
```
Comment (jakimfett) [inline LICENSE.md]: This license is mostly intended as a code and compiled binary license, not an art license. This section could use clarification, but more specific is probably not the direction to go. The less specific the language used (eg contributor rather than "person who submits a pull request via github") the better the license is able (to a certain extent) to protect the person using it.

File: LICENSE.md [hunk 2]
```diff
@@ -6,15 +6,19 @@ Version 2.0.3
 0. Definitions
 --------------
 
+**Source Code**: The preferred form of the work for making modifications to it.
+
+**Object code**: Any non-source form of a work.
```
Comment (jakimfett) [inline LICENSE.md]: What is the purpose of adding "object code"? What is the difference between object code and source code?

File: LICENSE.md [hunk 3]
```diff
@@ -26,13 +30,16 @@ Version 2.0.3
...
 
-**Author(s)**: Original programmer or programmers responsible for writing the Mod.
+**Author(s)**: Original programmer(s), texturer(s), and contributors responsible for creating, in part or in whole, the Mod.
 
 **Contributor**: Programmer or programmers who contribute code to the Mod, but are not the Author.
 
 **Codebase**: The Mod source code, complete with source history and Contributor records.
 
-**Distributor**: Anyone who distributes the compiled mod.
+**Distributor**: Anyone who distributes the mod, in source or object form.
```
Comment (jakimfett) [inline LICENSE.md]: I intentionally differentiated between the source form and the compiled form in the definition of "distributor", because technically using git as a version control system is distributing the source code. Also, I have no problem with someone handing my code to their friends...this license is (in part) intended to prevent people from re-hosting the compiled binaries and making money from it.

File: LICENSE.md [hunk 4]
```diff
@@ -47,12 +54,11 @@ This license is separate from the license of any Mod it depends on and does not
 --------
 
 The User accepts the following liability:
- - The Mod is provided 'as is' with no warranties, implied or otherwise.
- - The Author is not responsible for dragon, troll, pirate, or ninja attacks.
```
Comment (jakimfett) [inline LICENSE.md]: Absolutely do not remove this line. Refer to [PR#13](https://github.com/jakimfett/MMPLv2/pull/13) for the discussion of why.

File: LICENSE.md [hunk 5]
```diff
@@ -47,12 +54,11 @@ This license is separate from the license of any Mod it depends on and does not
...
+ - The Mod is provided as-is with no warranties, implied or otherwise.
  - Slaying any and all mythological creatures is the sole responsibility of the User.
- - The Author takes no responsibility for any damages incurred from the use or misuse of the Mod.
+ - The Author takes no responsibility for any damages incurred from the use or misuse of the Mod, direct or otherwise.
```
Comment (jakimfett) [inline LICENSE.md]: What's the point of adding "direct or otherwise" here? It clarifies nothing, basically just saying "any damages of any kind" again.

File: LICENSE.md [hunk 6]
```diff
@@ -64,7 +70,7 @@ The User may not limit access to the Mod in any way, except as provided by the G
 4. Right to examine
 --------
 
-The User may decompile compiled binaries of the Mod.  
+The User may decompile or otherwise translate the object code of the Mod.
```
Comment (jakimfett) [inline LICENSE.md]: What is the purpose of this change?

File: LICENSE.md [hunk 7]
```diff
@@ -73,19 +79,19 @@ The User may examine the Mod's source code.
 The Author reserves the right to distribute the Mod.  
 The Author may designate a person or organization as a Distributor.  
 A Distributor is granted the right to distribute the Mod.  
-The Author may require that a Distributor provides a direct link to the Codebase.  
+The Author may require that a Distributor provides a direct link to the Mod.
```
Comment (jakimfett) [inline LICENSE.md]: I'm interested in your thought process relating to this change. Please explain what the difference between "codebase" and "mod" does, and why it's a good thing.

File: LICENSE.md [hunk 8]
```diff
@@ -73,19 +79,19 @@ The User may examine the Mod's source code.
 The Author reserves the right to distribute the Mod.  
 The Author may designate a person or organization as a Distributor.  
 A Distributor is granted the right to distribute the Mod.  
-The Author may require that a Distributor provides a direct link to the Codebase.  
+The Author may require that a Distributor provides a direct link to the Mod.  
 The User may not distribute the Mod without being granted distribution rights from the Author.  
-The Author may revoke distribution rights from any Distributor.  
+The Author may revoke distribution rights from any Distributor without prior notice.
```
Comment (jakimfett) [inline LICENSE.md]: Unnecessary. Nowhere does this license say anything about giving notice. No need to start mentioning it now.

File: LICENSE.md [hunk 9]
```diff
@@ -73,19 +79,19 @@ The User may examine the Mod's source code.
...
+The Author may require that a Distributor provides a direct link to the Mod.  
 The User may not distribute the Mod without being granted distribution rights from the Author.  
-The Author may revoke distribution rights from any Distributor.  
+The Author may revoke distribution rights from any Distributor without prior notice.  
 
 6. Right to contribute
 --------
 
-The User may "fork" the Codebase.  
+The User may fork the Mod or any Software associated.
```
Comment (jakimfett) [inline LICENSE.md]: No. This license intentionally avoids making any statements about what the user can or can't do with other things. Remove this, please, unless you can provide a good reason why it should be added.

Additionally, you don't fork a mod...you fork the codebase. So this change is both overly intrusive (defining what the user can/can't do outside of the licensed code) and incorrect.

File: LICENSE.md [hunk 10]
```diff
@@ -73,19 +79,19 @@ The User may examine the Mod's source code.
...
 --------
 
-The User may "fork" the Codebase.  
+The User may fork the Mod or any Software associated.  
 The User may compile the Codebase.  
 The User may submit contributions to the Author for inclusion in the Mod.  
 The User grants the Author all rights to any contribution.  
-The User renounces all rights to their contribution except as specified under this license.  
-The User retains the right to re-use code they've written.  
+The User renounces all rights to their contribution except as specified otherwise by the Author.
```
Comment (jakimfett) [inline LICENSE.md]: No. If anything, the whole "except as specified" bit should be removed. The option for the author to provide an additional amending contributor license agreement is sufficient. Legally, it's better to make a blanket statement then define the specific exceptions, rather than defining each case specifically.

File: LICENSE.md [hunk 11]
```diff
@@ -73,19 +79,19 @@ The User may examine the Mod's source code.
...
 
-The User may "fork" the Codebase.  
+The User may fork the Mod or any Software associated.  
 The User may compile the Codebase.  
 The User may submit contributions to the Author for inclusion in the Mod.  
 The User grants the Author all rights to any contribution.  
-The User renounces all rights to their contribution except as specified under this license.  
-The User retains the right to re-use code they've written.  
+The User renounces all rights to their contribution except as specified otherwise by the Author.  
+The User retains the right to use any contrbutions they've made.
```
Comment (jakimfett) [inline LICENSE.md]: No. Absolutely not. Contributed code is code that the contributor has GIVEN to the author. They retain the right to use it, but they are completely giving up their ownership of that specific bit of code.

A clause like this one is what allowed a single contributor to eviscerated the Bukkit project, and I **will not** allow language like this into the license.

File: LICENSE.md [hunk 12]
```diff
@@ -131,5 +137,5 @@ The Author must make the Mod source freely and publicly available.
 --------
 
 The Author may grant a User the status of Contributor or Author.  
-The Author retains the right to the Mod name.  
+The Author retains the right to the Mod's name, Assets, and all other .
```
Comment (jakimfett) [inline LICENSE.md]: "and all other"...?

Maybe go ahead and finish this statement?

As a side note, it'd be better to list additional rights on additional lines.

File: LICENSE.md [hunk 13]
```diff
@@ -6,15 +6,19 @@ Version 2.0.3
...
 
-**Mod**: Software which modifies the Game, in any form (source, compiled binary, etc).
+**Mod**: Software which modifies the Game, in any form (source code, object code, etc).
 
 **Asset**: Graphic, audio, data file, or other medium used by the Mod.
 
 **User**: Anyone who that interacts with the Game or the Mod in any way.
 
-**Dependency**: Software required for the Mod to execute, compile, or otherwise "work" correctly.
+**Dependency**: Software required for the Mod to execute, compile, or otherwise operate correctly.
```
Comment (jakimfett) [inline LICENSE.md]: Given that "work" and "operate" are synonyms, I'm curious why you want to change this. Please provide a reason.

**Comment by jakimfett** [comment]
Given the large number of concerns I have with the wording changes, I'm going to need you to go through and provide specific reasoning for **each and every one** of these changes, @minerguy31.

In the future, please spell check your contributions, and double check to make sure that you actually finished writing each bit that you're adding.

File: LICENSE.md (hunk 2 above)
Comment (Strikingwolf) [inline LICENSE.md]: By this he means binaries. I agree there should be a better definition of this, but this is not the best definition of it. This goes along with below, For example, what if the language is not a compiled language, that means that author could argue that distributing the source is distributing the object code, thus blocking git and all else...tricky to get that right

File: LICENSE.md (hunk 6 above)
Comment (Strikingwolf) [inline LICENSE.md]: I guess translating to different languages....

**Comment by jakimfett** [comment]
@minerguy31 please take a few minutes to respond to my questions, I'd like to get this PR moving.

**Comment by jakimfett** [comment]
@minerguy31 closing this pull request due to lack of response in over 2 months. Please open a new pull request after reading my comments.
